uint8 CHILD_AREA_EXTERNAL = 0
uint8 CHILD_AREA_OBSERVATION = 1
uint8 CHILD_AREA_INTERACTION = 2

Header  header
float64 child_observation_distance
uint8   child_area
float64 robot_bounds_proximity
